schema/base.graphqls

```diff
@@ -46,13 +46,61 @@ type SignalAggregations {
   Timestamp of the aggregated data.
   """
   timestamp: Time!
+  """
```

Comment:
Could we be a little more specific about what happens to the location? I know no one reads these strings but